src/parser.rs

```diff
@@ -110,6 +112,12 @@ where
             .filter(stdlib::Where)
     }
 
+    /// Sets the parse mode to lax.
+    pub fn in_lax_mode(mut self) -> Self {
+        self.mode = ParseMode::Lax;
+        self
+    }
```

Comment:
I think I'd rather re-export the num and accept it.

Reply:
I'll update, I did it this way since there's only 2 options currently, 1 of which is the default.  So exposing a function allowed us to now have to worry about exposing the enum to the user.